Mage/src/main/java/mage/target/common/TargetSuspendedCard.java

```diff
@@ -0,0 +1,114 @@
+package mage.target.common;
+
+import mage.MageObject;
+import mage.abilities.Ability;
+import mage.cards.Card;
+import mage.constants.Zone;
+import mage.filter.Filter;
+import mage.filter.common.FilterSuspendedCard;
+import mage.game.Game;
+import mage.target.TargetImpl;
+
+import java.util.HashSet;
+import java.util.Set;
+import java.util.UUID;
+
+/**
+ * @author emerald000
+ */
+public class TargetSuspendedCard extends TargetImpl {
```

Comment:
This won't be needed at all. Use `TargetCardInExile` with the suspended card filter (and `withNotTarget(true)` for this application).

Reply:
if now unused, please remove